contribs/src/main/java/com/netflix/conductor/contribs/metrics/DatadogMetricsConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package com.netflix.conductor.contribs.metrics;
+
+import com.netflix.spectator.api.Spectator;
+import com.netflix.spectator.micrometer.MicrometerRegistry;
+import io.micrometer.core.instrument.MeterRegistry;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Metrics Datadog module, sending all metrics to a Datadog server.
+ * <p>
+ * Enable in config: conductor.metrics-datadog.enabled=true
+ * <p>
+ * Make sure your dependencies include both micrometer-registry-datadog & spring-boot-starter-actuator
+ */
+@ConditionalOnProperty(value = "management.metrics.export.datadog.enabled", havingValue = "true")
```

Comment:
It would be clearer to use the same property `conductor.metrics-datadog.enabled` here as well as to enable the management endpoint.